docs/semgrep-cloud-platform/github-pr-comments.md

```diff
@@ -21,60 +23,116 @@ Object.entries(frontMatter).filter(
+
+Semgrep can create **pull request (PR) comments** in your GitHub repository. These comments inform your developers of **findings**, such as security issues, in their PRs. Semgrep can also provide remediation tips or code fixes that your developers can click to **commit** into their code directly.
 
 :::info Prerequisites
-* Pull request (PR) comments can only be enabled through Semgrep Cloud Platform (SCP). [Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to set up PR comments.
-* To receive alerts and notifications, you must [add or onboard a project](/semgrep-code/getting-started/#option-b-adding-a-repository-from-github-gitlab-or-bitbucket) (repository) to Semgrep Cloud Platform for scanning.
+- Pull request (PR) comments can only be set up through Semgrep Cloud Platform (SCP). [<i class="fas fa-external-link fa-xs"></i> Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to set up PR comments.
+- You must connect your GitHub organization (org) to Semgrep.
+    - For **GitHub Cloud**: See [<i class="fa-regular fa-file-lines"></i> Adding an organization](/semgrep-cloud-platform/getting-started/#adding-an-organization).
+    - For **GitHub Enterprise Server**: [<i class="fa-regular fa-file-lines"></i> Integrate Semgrep with GitHub Enterprise](/semgrep-cloud-platform/scm/#integrating-semgrep-cloud-platform-with-github-enterprise-or-gitlab-self-managed) and if you have a VPN or firewall, [set up your allowlists](/semgrep-cloud-platform/scm/#receiving-pr-or-mr-comments-in-your-vpn-or-on-premise-scm).
+- You must add or onboard a Project (repository) to SCP and it must complete at least one full scan successfully.
 :::
+## Conditions for PR comment creation
 
-Pull request comments are created when:
+PR comments appear for the following types of scans and under certain conditions:
 
-1. Semgrep finds a result in CI.
-2. The Semgrep GitHub App has permissions to post inline PR comments.
+<CommentTriggers />
 
 Automated comments on GitHub pull requests are displayed as follows:
 
 ![Screenshot of a GitHub PR comment](/img/semgrep-pull-request.png#bordered)
 **Figure** An inline GitHub pull request comment.
 
-[Semgrep Cloud Platform](https://semgrep.dev/manage) uses the permissions requested by [the Semgrep GitHub App](https://github.com/marketplace/semgrep-dev) to leave PR comments. You can verify that you have granted these permissions by visiting either `https://github.com/organizations/<your_org_name>/settings/installations` or `https://github.com/organizations/<your_org_name>/<your_repo_name>/settings/installations`.
+## Confirm account connection and access
+
+### Confirm your Semgrep account's connection to GitHub
+
+Confirm that you have the correct connection and access:
+
+1. In your SCP account, click **Settings > Source code managers**.
+2. Check that an entry for your GitHub org exists and is correct.
```

Comment:
do we want to avoid using org the way we avoid repo?

Reply:
Fortunately, it is spared from an entry in the [style guide word list](https://developers.google.com/style/word-list). However, I do open with "organization (org)" in both GitHub (line 54) and GitLab (line 33).